### PR TITLE
Add support for new incentive types

### PR DESCRIPTION
**Pull request by cjohnsbrown**
## Description
[Context](https://docs.google.com/document/d/1Lqwx0fqBhoqfHuE1S_toJhhCy3ZULT3j15EFcGDoz0Q/edit#heading=h.1405zbe4qx7l)

Add new incentive types to UI for lawn equipment and Ebikes

## Test Plan
* `yarn test`
* `yarn cypress run`
* Manually tested with dev server

### CODE REVIEW

**Comment by veekas** [comment]
I pulled up an Electric Empire Association zip code and was looking for a lawn mower incentive as expected, and didn't see anything... but that could be user error on my part. Can you share what zip codes you used to test out these new incentives?

**Comment by cjohnsbrown** [comment]
> I pulled up an Electric Empire Association zip code and was looking for a lawn mower incentive as expected, and didn't see anything... but that could be user error on my part. Can you share what zip codes you used to test out these new incentives?

I tried 81321 in the vercel preview

Screenshot 2024-08-21 at 12 21 01 PM

File: src/i18n/strings/es.ts [hunk 1]
```diff
@@ -205,4 +205,8 @@ export const templates = {
   sfa7338035e1ef173: `Alquilar o poseer`,
   sfc7214f623fe475d: `Selecciona la empresa a la que paga su factura de electricidad.`,
   sfe16afc784bb9d76: `Techo solar`,
+  s8ff568239ce4b254: `Lawn Care`,
+  s81f5c9b4a6da995f: `Lawn`,
+  sab856460d5aed19e: `electric outdoor equipment`,
+  sca0eca5aae5454cd: `an E-bike`,
```
[the remark this answers is not in the document]
Comment (cjohnsbrown) [reply src/i18n/strings/es.ts]: changed this to lower case

File: .gitignore [hunk 2]
```diff
@@ -29,3 +29,5 @@ yarn-error.log*
 # cypress
 cypress/videos/*.mp4
 cypress/screenshots/*
+
+.idea/
```
[the remark this answers is not in the document]
Comment (cjohnsbrown) [reply .gitignore:33]: I started using webstorm for a personal project and I like it a lot

**Review by veekas — APPROVED** [review]
Looks great, thanks!

**Pull request merged**